src/QuadForm/Herm/Genus.jl

```diff
@@ -1241,26 +1241,26 @@ Return the rank of any hermitian lattice with global genus symbol `G`.
 rank(G::GenusHerm) = G.rank
 
 @doc Markdown.doc"""
-    scale(G::GenusHerm) -> NfOrdFracIdl
+    scale_norm(G::GenusHerm) -> NfOrdFracIdl
```

Comment:
I think that this is not the norm of the scale. It is $s(L) \cap K$ where $E/K$ and $K$ is the fixed field. 
So a more appropriate name could be `scale_in_fixed_field`. 
But certainly there should be a function `scale(L)` returning the honest scale of `L` as a fractional ideal in `E`.

Reply:
I tried with different examples and I actually got the norm of the scale. I guess the point is that the valuation computed in the genus symbol are coming from taking the support of the norm of the discriminant of the lattice. I thought at first it was the intersection but realised not.

For lattices, we have such a scale function that returns a fractional ideal of `E` indeed.

Reply:
I still disagree. 
Consider $E= \mathbb Q [\sqrt 5]$ and $L$ with gram matrix $[3]$.
The scale is $3 \mathcal O_E$ the norm of the scale is $9 \mathbb Z$. 
But the $s_i$ in thegenus symbol should be $1$. So I think your function would return 
$3 \mathbb Z$?
```
julia> D = matrix(E, 1, 1, [3])
[3]

julia> L = hermitian_lattice(E, gram = D)
Hermitian lattice of rank 1 and degree 1
over
Relative maximal order with pseudo-basis (1) * 1//1 * <1, 1>, (b) * 1//1 * <1, 1>

julia> scale(L)
Fractional ideal of
Relative maximal order with pseudo-basis (1) * 1//1 * <1, 1>, (b) * 1//1 * <1, 1>
with basis pseudo-matrix
(1//1 * <3, 3>) * [1 0]
(1//1 * <3, 3>) * [0 1]

julia> genus(L)
Global genus symbol over Relative number field with defining polynomial t^2 + 5
 over Number field over Rational Field with defining polynomial x - 1
with local genus symbols
<3, 3> => (1, 1, +)
<2, 2> => (0, 1, -, 0)
<5, 5> => (0, 1, -)
and signatures
1.0 => 0
```
Anyways what happens depends on whether the prime is ramified or not.

Reply:
So I am not sure your function makes sense.

Reply:
Thanks for pointing out your example, so now it appears clearer that my function only detect by what I need to rescale for having a "nice" `representative` function.

Reply:
Yea, internal makes more sense.